src/components/Sidebar/Stepper.tsx

```
@@ -15,19 +19,9 @@ export const Stepper: FC<StepperProps> = props => {
-          // @ts-ignore
-          logoColor='#073387'
-          // @ts-ignore
-          wordColor='#131313'
-          // @ts-ignore
-          betaColor='#073387'
-          width={betaLogo ? 152 : 131}
-          height={betaLogo ? 22 : 24}
-          onClick={onLogoClick}
-          wrapperCls={classes('header-logo', logoCls)}
-        />
+        <span className={classes('header-logo', logoCls)} onClick={onLogoClick}>
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)